packages/lambda-xyz/src/__test__/ping.test.ts

```diff
@@ -0,0 +1,14 @@
+import { LambdaHttpResponseAlb } from '@basemaps/lambda-shared';
+import * as o from 'ospec';
+import ping from '../ping';
+
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+o.spec('ping', () => {
```

Comment:
this is tested in the file above?

Reply:
yep I wanted your opinion if you liked tests per file; seems you don't (like me) so I merge these tests